net-libs/tox/tox-0.2.7.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils systemd user
+
+MY_P="c-toxcore-${PV}"
+DESCRIPTION="Encrypted P2P, messaging, and audio/video calling platform"
+HOMEPAGE="https://tox.chat"
+SRC_URI="https://github.com/TokTok/c-toxcore/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0/0.2"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+av daemon ipv6 log-debug log-trace +no-log static-libs test"
+
+REQUIRED_USE="^^ ( no-log log-trace log-debug )"
+
+RDEPEND="
+	av? ( media-libs/libvpx:=
+		media-libs/opus )
+	daemon? ( dev-libs/libconfig )
+	>=dev-libs/libsodium-0.6.1:=[asm,urandom]"
+DEPEND="${RDEPEND}
+	>=dev-util/cmake-2.8.6
```

Comment:
cmake-utils should take care of the dep, doesn't it?

Reply:
It does seem to, though for good measure I set CMAKE_MIN_VERSION ?= 2.8.6. on my local repo.

Reply:
The default is much higher, so this is generally a bad idea:

```
: ${CMAKE_MIN_VERSION:=3.9.6}
```

It means you might actually accept CMake that is too old for the eclass to work with.

Reply:
Alright, I removed the variable. Is this getting into "implicit dependencies"?

Reply:
No. It's about eclass-defined dependencies.